Add AWS WAF native (#7511)
wazuh/wazuh-documentation: source/cloud-security/amazon/services/supported-services/waf.rst

Reviewer: Given that the `Amazon configuration with Kinesis` will be deprecated, we should modify the `Wazuh configuration` section to contain the `waf_acls` parameter as part of the example.

`Amazon WAF <https://aws.amazon.com/waf/>`_ is a web application firewall that helps protect your web applications from common web exploits that could affect application availability, compromise security, or consume excessive resources. ``AWS WAF`` gives you control over which traffic to allow or block to your web applications by defining customizable web security rules. 

Amazon configuration native integration
---------------------------------------

#. :doc:`Create a new </cloud-security/amazon/services/prerequisites/S3-bucket>` S3 bucket. (If you want to use an already created one, skip this step).

> #. Go to Services > Security, Identity, & Compliance > WAF & Shield

Reviewer 2: ```suggestion
#. Go to Services > Security, Identity, & Compliance > WAF & Shield:
```

    .. thumbnail::  /images/aws/waf-native-1.png
      :align: center
      :width: 70%

> #. Click *Create web ACL*

Reviewer 2: ```suggestion
#. Click on *Create web ACL*:
```

    .. thumbnail::  /images/aws/waf-native-2.png
      :align: center
      :width: 70%

> #. Put a name to your web ACL and click on the *Next* button at the bottom of the page:

Reviewer 2: ```suggestion
#. Choose a name for your Web ACL and click on *Next*:
```

The next button is not shown in the image, perhaps we could do a little of zoom out and include it to be clear.

    .. thumbnail::  /images/aws/waf-native-3.png
      :align: center
      :width: 70%

> #. Click *Add rules* to *add managed rule groups* or *add my own rules and groups* and click on the *Next* button at the bottom of the page.:

Reviewer 2: ```suggestion
#. Click on *Add rules* and select which type of rule to add, then click on *Next*:
```

We are instructing the user to add a rule but we do not specify which or what to put in the form, is this expected?

Author: Regarding the rules to implement, it was decided to keep it simple, showing the user where to add them because there are many ways to implement them.

Under "Add managed rule groups":

AWS managed rule groups
Cloudbric Corp. managed rule groups
Cyber Security Cloud Inc. managed rule groups
...
And under "Add my own rules":

Rule JSON editor
Rule visual editor
...
Therefore, adding a specific way of doing it didn't seem like the best option.

    .. thumbnail::  /images/aws/waf-native-4.png
      :align: center
      :width: 70%

> #. Set rule priority and click on the *Next* button at the bottom of the page:

Reviewer 2: ```suggestion
#. Set the rules priority and click on *Next*:
```

    .. thumbnail::  /images/aws/waf-native-5.png
      :align: center
      :width: 70%

> #. Put a name to your CloudWatch metric and click on the *Next* button at the bottom of the page:

Reviewer 2: ```suggestion
#. Choose a name for your CloudWatch metric and click on *Next*:
```

Where did we create this metric? If it is required we should instruct the user how to do it.

    .. thumbnail::  /images/aws/waf-native-6.png
      :align: center
      :width: 70%

> #. Review the ACL web and click on the *Create web ACL* button at the bottom of the page to create it

Reviewer 2: ```suggestion
#. Review the Web ACL fields and click on *Create web ACL*  to create it:
```
